README.md

```diff
@@ -1 +1,122 @@
-# My-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Welcome to my portfolio</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+**[Portfolio]** shows information about me and my projects.<br />
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ - HTML 
+ - CSS
+- JAVASCRIPT
+
+### Key Features <a name="key-features"></a>
+
+- **[Application has interactive buttons]**
+- **[Welcoming text and introducing myself]**
+- **[Link to my projects and social media accounts]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming Soon]()
```

Comment:
Kindly, I suggest you remove the live demo section in your Readme file if your project is not deployed yet. You can add it anytime you want when the live demo link is available. 😄